stompServer.js

```diff
@@ -479,7 +479,7 @@ var StompServer = function (config) {
 
   this.parseRequest = function(socket, data) {
     // check if it's incoming heartbeat
-    if (socket.heartbeatClock != null && data === Bytes.LF) {
+    if (socket.heartbeatClock !== undefined && data === Bytes.LF) {
```

Comment:
Another question:
This needs data to be exactly `Bytes.LF` to consider it as a heart-beat. As far as I saw normal message frames does count as well. You do not need to send "PING" messages when normal message arrived in that time-frame. This is the behavior using SockJs client.
As far as I see, it is the same [Stomp doc 1.2](https://stomp.github.io/stomp-specification-1.2.html#Heart-beating)

Beside this, it looks good! Thanks!

Reply:
Yeah! Totally forgot about that! Good catch! I will fix it at once 👍